src/sass/partials/layouts/_login.scss

```diff
@@ -0,0 +1,17 @@
+.login--wrapper {
+  margin: 0 auto;
+  position: relative;
+  width: 300px;
+}
+
+.login--title {
+  padding: .5em 0;
+}
+
+.login--fields {
+  display: block;
+  padding: .5em 0;
+  > * {
+    width: 100%;
+  }
+}
```

Comment:
Some of your files have new lines at the end, some do not. Please make sure you've got [Editorconfig](http://editorconfig.org/) installed and working